api/spec/src/paths/tags/single/delete.yaml

```diff
@@ -0,0 +1,9 @@
+parameters:
```

Comment:
I don't think we need anything besides POST for creating `tag` - I can't think of situation when we would want to delete a tag directly from db collection - why would user be even able to do that?
Unless we are talking about removing tag from a project - but then, it should be in a different directory and accept different parameters, namely both `projectId` and `tagId`